appcli/models.py

```diff
@@ -49,18 +52,17 @@ class ConfigSettingsGroup(NamedTuple):
     title: str
     settings: List[ConfigSetting]
 
+class Hooks(NamedTuple):
+    pre_configure_init: Callable[[CliContext], None] = lambda x: None
+    """ Hook to run before running configure. """
```

Comment:
be explicit in the comments for all of these:
`Hook to run before running "configure init"`